### ISSUE TITLE
Mongoose deprecation warning

### ISSUE THREAD

**Issue by cheng-alvin** [Chillandchat/api#5]
A deprecation warning was found on the API, this deprecation warning will be logged whenever the server starts. The warning goes as such:
```
(node:49537) [MONGOOSE] DeprecationWarning: Mongoose: the `strictQuery` option will be switched back to `false` by default in Mongoose 7. Use `mongoose.set('strictQuery', false);` if you want to prepare for this change. Or use `mongoose.set('strictQuery', true);` to suppress this warning.
(Use `node --trace-deprecation ...` to show where the warning was created)
```

Action should be taken in order to update the code and remove the deprecated feature(s). Deprecated mongoose features may be unreliable and cause more issues in the future.

**Comment by cheng-alvin** [comment]
As NeNaD mentioned on [stackoverflow.com](https://stackoverflow.com), "When strict option is set to true, Mongoose will ensure that only the fields that are specified in your Schema will be saved in the database, and all other fields will not be saved (if some other fields are sent).Right now, this option is enabled by default, but it will be changed in Mongoose v7 to false by default. That means that all the fields will be saved in the database, even if some of them are not specified in the Schema model."

So to summarize, he mentioned that this protection measure to prevent missing values will be **DISABLED** from mongoose v7.0.0. And we need to set it manually by hand when connecting. He left a snippet of code that reads: 
```js
mongoose.set('strictQuery', true);
``` 
This code above will set the ```strictQuery``` option to true.

So in mongoose v7 this will prevent the ```strictQuery``` option to default to ```false```.

**Comment by cheng-alvin** [comment]
All action we need to take is to add the line of code to the line above the```mongoose.connect()``` line

**Comment by cheng-alvin** [comment]
This issue's progress will be tracked at #6
